### PR TITLE
Added site down time policy

### PR DESCRIPTION
**Pull request by jabelone**
This policy aims to avoid the dissapointment of showing up on site to find an area that is unable to be used by documenting requirements around notifying members of area down time.

### CODE REVIEW

**Comment by jhogendorn** [comment]
if this is going to be a policy, it should standalone and not be just referring to electrical works. it should also be expanded to include tools/machines.

File: admin/sitemodselectrical.md [hunk 1]
```diff
@@ -10,9 +10,18 @@ HSBNE Inc. has certain restrictions that apply at our current property. This pol
 
 * Any references made to **“site”**, **“on site”**, etc. refer to the HSBNE porthack campus located in Hamilton.
 * **Site modification** - is any alteration to walls, doors, fixtures, or any other part of our site that is permanent or cannot be easily reversed by any member.
+* **Area Downtime** - any modification to a HSBNE area (such as a cause area or other) that prevents a member from temporarily using any part of the area.
 * **Site Manager** - is someone who has been appointed by the landlord to act on behalf of them to help manage or maintain the site.
 * **Written notice** - a formal notice that can be delivered via email, discord or in person.
 
+# Area Downtime
+HSBNE Members expect that the site is available for use 24/7 as explained during their induction and on our website. This policy aims to avoid the dissapointment of showing up on site to find an area that is unable to be used. This is especially important for members who have to travel far to get to HSBNE or who make special arrangements/plans.
```
Comment (jhogendorn) [inline admin/sitemodselectrical.md:18]: have a reasonable expectation, but there is no obligation for hsbne to guarantee this. if things are going to be unavailable, its worth laying out whos responsible for communicating that and where.

File: admin/sitemodselectrical.md [hunk 2]
```diff
@@ -10,9 +10,18 @@ HSBNE Inc. has certain restrictions that apply at our current property. This pol
...
+
+* Members must be notified in writing of all changes to an area of HSBNE that may, or will, cause Area Downtime.
```
Comment (jhogendorn) [inline admin/sitemodselectrical.md:20]: this should be more specific, ie the official communications method of the group

**Comment by jabelone** [comment]
It's not just referring to the electrical works policy - it's an entirely separate heading so not sure where you get that impression from. This entire document is called "site modifications and electrical work" so I believe this is a good place for it and creating a separate policy document for what is essentially a paragraph is unnecessary.

File: admin/sitemodselectrical.md [hunk 3]
```diff
@@ -10,9 +10,18 @@ HSBNE Inc. has certain restrictions that apply at our current property. This pol
...
+* Other than in exceptional circumstances, at least 48 hours prior notice is required for Area Downtime notifications.
+* Members must be notified via the official communication method of the association of all unexpected or prolonged Area Downtime immediately (ie as soon as it's found out that Area Downtime is happening or will be extended).
+* All changes to an area of HSBNE that may, or will, cause Area Downtime should be planned to happen during times of low site use where reasonably practical, taking into account volunteer availability.
+
```
Comment (jhogendorn) [inline admin/sitemodselectrical.md:24]: I'd like to see all this to be clearer about who is responsible for reporting, and if it applies to machines/tools.

File: admin/sitemodselectrical.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ HSBNE Inc. has certain restrictions that apply at our current property. This pol
 # Area Downtime
 HSBNE Members expect that the site is available for use 24/7 as explained during their induction and on our website. This policy aims to avoid the dissapointment of showing up on site to find an area that is unable to be used. This is especially important for members who have to travel far to get to HSBNE or who make special arrangements/plans.
 
-* Members must be notified in writing of all changes to an area of HSBNE that may, or will, cause Area Downtime.
+* The member in charge of any modification to a HSBNE area must make an anouncement via the official communication method of all changes that may, or will, cause Area Downtime.
```
Comment (jhogendorn) [inline admin/sitemodselectrical.md:20]: Is this the person performing the change or is it the cause leader for that area? This is the sort of thing a RACI is made for. Do we need to provide provisions for downtimes caused by third parties? ie the exec isnt in charge of say, water works performed by cbre, but we try and pass on the notification to members, typically by email.

**Comment by jabelone** [comment]
This was approved at a general meeting: https://wiki.hsbne.org/meetings/general/2021/20210323

**Pull request merged**